ml-lifecycle/feature_store/FS_demo.ipynb

```diff
@@ -24,7 +24,6 @@
    "source": [
```

Comment:
<p><code>Feature Store</code> rather than <code>FeatureStore</code>. ﻿Just do a find and replace, there are more below as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2814/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>